### DISCUSSION TITLE
Multi Select can be create multiple items in a single submission

### DISCUSSION THREAD

**Discussion by mazipan** [mantinedev/mantine#3723] [Feature requests]
Basically it's like https://www.npmjs.com/package/react-tag-input

And we can pass multiple items using a separator, e.g: comma or space character.
Currently, it's not possible.

![Screen Shot 2023-03-09 at 10 29 01](https://user-images.githubusercontent.com/7221389/223909316-6f6dbfb6-877f-4246-9b9e-9f07d00164ca.png)

**Comment by rtivital** [comment]
This feature will not be added to multiselect. See TagInput, maybe it will be more suitable for your use case – https://github.com/mantinedev/mantine/pull/3674
